packages/react-effect/src/Effect.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {METHOD_NAME, Extractable} from './extractable';
+
+interface Props {
+  kind?: symbol;
+  serverOnly?: boolean;
+  clientOnly?: boolean;
+  perform(): void;
+}
+
+export default class Effect extends React.PureComponent<Props>
+  implements Extractable {
```

Comment:
I am so not crazy about this line break that I would suggest moving the `export default Effect` out to the bottom.

Reply:
Meh, just let prettier do its thing :P